### ISSUE TITLE
(aws_cdk.aws_apigatewayv2_integrations.LambdaWebSocketIntegration): produces wrong integration_uri

### ISSUE THREAD

**Issue by jrobbins-LiveData** [aws/aws-cdk#13679]
`LambdaWebSocketIntegration` (per the example [here](https://github.com/aws/aws-cdk/blob/fe1c8393e0840fb273c4a5f325cb3cebc784bf4b/packages/%40aws-cdk/aws-apigatewayv2-integrations/README.md#websocket-apis)) seems designed to plug a lambda handler into a WebSocket API Gateway.  But it produces the wrong `IntegrationUri`.

### Reproduction Steps

see example [here](https://github.com/aws/aws-cdk/blob/fe1c8393e0840fb273c4a5f325cb3cebc784bf4b/packages/%40aws-cdk/aws-apigatewayv2-integrations/README.md#websocket-apis)

### What did you expect to happen?

Expected a working API Gateway.

### What actually happened?

Ended up with a broken API Gateway because the generated `IntegrationUri` is wrong.

### Environment

  - **CDK CLI Version  :** 1.94.1 (build 60d8f91)
  - **Framework Version:**
  - **Node.js Version:** 
  - **OS               :**
  - **Language (Version):** 

### Other

The `IntegrationUri` produced is

```
            "IntegrationUri": "arn:aws:lambda:us-east-2:$$ACCOUNT ID redacted$$:function:sampleContainerFunction",
```

but a correct one would be

```
            "IntegrationUri": "arn:aws:apigateway:us-east-2:lambda:path/2015-03-31/functions/arn:aws:lambda:us-east-2:$$ACCOUNT ID redacted$$:function:sampleContainerFunction/invocations",
```

For documentation on this "invocations" format for the `IntegrationUri`, please see this [doc](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-websocket-api-integration-requests.html).

The correct AWS CLI command `create-integration` shows the format:

```
aws apigatewayv2 create-integration --api-id aabbccddee --integration-type AWS_PROXY --integration-method POST --integration-uri arn:aws:apigateway:us-east-1:lambda:path/2015-03-31/functions/arn:aws:lambda:us-east-1:123412341234:function:simpleproxy-echo-e2e/invocations
```

--- 

This is :bug: Bug Report

**Comment by jrobbins-LiveData** [comment]
The underlying [implementation](https://github.com/aws/aws-cdk/blob/fe1c8393e0840fb273c4a5f325cb3cebc784bf4b/packages/%40aws-cdk/aws-apigatewayv2-integrations/lib/websocket/lambda.ts#L41) of `LambdaWebSocketIntegration` seems to be the source of the problem. 

```
    return {
      type: WebSocketIntegrationType.AWS_PROXY,
      uri: this.props.handler.functionArn,
    };
```

Instead of `this.props.handler.functionArn`, the code needs to construct the correct uri.

In Python (sorry, that's the language I'm using), the constructor for the correct uri would look like

```
        lambda_invocation_arn = Stack.format_arn(
            sep=":",
            service="apigateway",
            resource="lambda",
            resource_name=f"path/2015-03-31/functions/{handler.function_arn}/invocations"
        )
```

In other words, the functionArn has to be embedded in a special wrapper.  If you create a WebSocket API Gateway integration via the AWS CLI and then inspect it, you will see that this is the correct format.

**Comment by nija-at** [comment]
cc @ayush987goyal

**Comment by ayush987goyal** [comment]
@jrobbins-LiveData @nija-at The current `IntegrationUri` works from functionality point of view. However it breaks the API Gateway console experience. Therefore it was not caught while initial implementation. I will make the change to fix this.

Thanks a lot for reporting!

**Comment by jrobbins-LiveData** [comment]
@ayush987goyal , I didn't realize it worked!  I was "verifying" my deployment by using the API Gateway console, which is what led me to my false conclusion.  Sorry about that!
